File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-14/source/2026-07-14-reliability-runbook-alert-triage-automation.py
"""Northstar Transit Cloud; Checkout SVC-17; 2026-07-13; error budget 2.4%; INC-260713."""
SERVICE = "Checkout SVC-17"
INCIDENT = "INC-260713"
TOPIC = "alert_triage"
SPINE_IDS = ['org.ntc', 'svc.checkout.svc17', 'rel.inc260713.timeline', 'date.2026-07-14', 'term.no-kio-claim']
FINGERPRINT = "823dda1c6e4fb3d2"

def alert_level(queue_depth: int) -> str:
    return 'critical' if queue_depth >= 24000 else 'warning' if queue_depth >= 12000 else 'normal'

File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-14/source/2026-07-14-reliability-runbook-alert-triage-automation.rs
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
pub const TOPIC: &str = "alert_triage";
pub const FINGERPRINT: &str = "bc6d9101aecd6527";
pub fn q07_is_critical(depth: u32) -> bool { depth >= 24_000 }
